__tests__/ExpensiMark-Markdown-test.js

```diff
@@ -33,7 +33,7 @@ test('Test strikethrough HTML replacement', () => {
 
 test('Test Mixed HTML strings', () => {
     const rawHTMLTestStartString = '<em>This is</em> a <strong>test</strong>. None of <h1>these strings</h1> should display <del>as</del> <div>HTML</div>.';
-    const rawHTMLTestReplacedString = '_This is_ a *test*. None of these strings should display ~as~ HTML.';
+    const rawHTMLTestReplacedString = '_This is_ a *test*. None of these strings should display ~as~ \nHTML\n.';
```

Comment:
the dot  and HTML  here should be a in  new so I modify it 


![Screen Shot 2022-03-24 at 8 43 51 AM](https://user-images.githubusercontent.com/21364901/159864018-9c168efd-dc48-43b2-8da6-3ef21b49d0e6.png)